atomic_reactor/schemas/config.json

```diff
@@ -695,6 +695,10 @@
               "description": "Url with denylist yaml file, which will be used to exclude sources from cachito ",
               "type": "string"
           },
+          "allowlist_cache": {
+              "description": "Url with allowlist yaml file, which allows usage of lookaside cache",
```

Comment:
I would prefer something like "lookaside_cache_allowlist", the current name sounds like something that would specify if/where/how to cache some allowlist

Reply:
ok